sqlmesh/dbt/basemodel.py

```diff
@@ -289,10 +294,27 @@ def sqlmesh_model_kwargs(self, context: DbtContext) -> t.Dict[str, t.Any]:
         jinja_macros = model_context.jinja_macros.trim(
             self.dependencies.macros, package=self.package_name
         )
+
+        model_attrs = {
+            macro.name.split(".")[1]
+            for macro in self.dependencies.macros
+            if macro.name.startswith("model.")
+        }
+
+        model: AttributeDict[str, t.Any] = AttributeDict(
```

Comment:
[Nit] Let's clarify it refers to a node in the manifest, eg. `model_node` or `model_manifest_node`.